modules/installing-rhv-creating-custom-vm.adoc

```diff
@@ -0,0 +1,179 @@
++
+. On the Manager, run the Ansible playbook. For example:
++
+----
+$ ansible-playbook rhcos_image.yaml -vvv
+----
+
+. In the Administration Portal, go to the *Storage* -> *Disks* page and find the disk name you specified in the playbook. For example, `rhcos_44-81_img-diskname`.
+. When the status of the disk is *OK*, the playbook has finished uploading the RHCOS image to the storage domain.
+
+.Attach the virtual disk with the RHCOS image to the virtual machine
+
+. Go to the *Compute* -> *Virtual Machines* page and click btn:[New].
+. In the *New Virtual Machine* -> *General* window that opens, use *Cluster* to select the {rh-virtualization} cluster where you plan to create the cluster.
+. Leave *Template* unchanged, with *Blank* selected.
+. Set *Operating System* to *Red Hat Enterprise Linux CoreOS*.
+. Leave *Optimized for* unchanged, with *Desktop* selected.
+. For *Name*, specify the name of the virtual machine. For example, `vmname`.
+. For *Instance Images*, click btn:[Attach].
+. In the *Attach Virtual Disks* window that opens, find the disk you specified in the playbook. For example, `rhcos_44-81_img-diskname`.
+. Click the radio button for this disk.
+. Select the *OS* checkbox for this disk. This makes the disk bootable.
+. Click btn:[OK] to save and close the *Attach Virtual Disks* window.
+
+.Continue configuring the virtual machine and create it
+
+. In the *New Virtual Machine* -> *General* window, for each NIC under *Instantiate VM network interfaces by picking a vNIC profile*, select a vNIC profile.
+. Go to the *New Virtual Machine* -> *General* window.
++
+[IMPORTANT]
+====
+In the following steps, if you increase the values of *Memory* or *Total Virtual CPU* beyond the amounts shown, multiply these new values by the number nodes the installation program creates. By default, the installation program creates seven nodes. Verify that your {rh-virtualization} cluster has the memory and vCPU resources to meet these new values. For example, increasing *Total Virtual CPUs* from *4* to *6* means that your cluster requires 42 available vCPUs instead of 28. These vCPUs must be available exclusively for the cluster, and not required for non-cluster computing.
+====
++
+. Set *Memory* to *16384 MB* or more. This value is equivalent to 16 GiB. Adjust this value according to the workload you expect on the compute machines.
+. Due to link:https://bugzilla.redhat.com/show_bug.cgi?id=1821215[bz#1821215], set *Physical Memory Guaranteed* to *8192 MB*. This value is equivalent to 8 GiB.
+. Set *Total Virtual CPUs* to *4* or more.
+. Adjust *Cores per Virtual Socket* to *4* or more so it matches the *Total Virtual CPUs* setting.
+. Press btn:[OK] to save and close the *New Virtual Machine*.
++
+The new virtual machine appears in the *Compute* -> *Virtual Machines* window. Because the virtual machine is not starting, it should appear quickly.
```

Comment:
I'd rewrite to avoid "should."

> ! should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as
appropriate.